stage0/gc.go

```diff
@@ -55,8 +55,12 @@ func GC(pdir string, uuid *types.UUID) error {
 	if debugEnabled {
 		args = append(args, "--debug")
 	}
+	if localConfig != "" {
+		args = append(args, "--local-config", localConfig)
```

Comment:
This is the exact same argument we pass to stage1/init, so I thought I'd mirror it. The root of the problem was that the Networking object had this set properly on Create but not on Load.